Deployment/Mirantis-Guidance.md

```diff
@@ -0,0 +1,274 @@
+
+Documentation:
+- [Configuring placement constraints for Service Fabric services](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-resource-manager-configure-services#placement-constraints)
+- [Service Fabric application upgrade](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-application-upgrade)
+- [Stateful service replica set size configuration](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-best-practices-replica-set-size-configuration)
+
+## Scenario 2: Customer is hosting Azure Service Fabric Node Type using deprecated Windows Server images with-Containers, and does host containers in Docker as part of their applications
+
+In general, the steps are as follows:
+1. Acquire licenses from Mirantis directly.
+   [Mirantis Enterprise Support](https://www.mirantis.com/docker-engine-enterprise-support/ )
+2. Install MCR on new node type. There are three options to install the container runtime:
+   1. Custom Script VM Extension on a new node type with standard Azure OS image without container support. 
+   Please find the guidance to install MCR via Custom Script VM Extension in the respective scenario.
+   The Mirantis installer needs a machine restart to work during the post-deployment. This can potentially delay other operational processes, as scaling, node repair, reimage can take longer. The installer also checks if the latest version is installed, this can be disabled in the provided script.
+   2. Create a new node type with the Azure Marketplace OS image provided by Mirantis with pre-installed container runtime. This image will be released in September 2022.
+   3. Create a custom OS image to pre-install MCR. Please find guidance below in the documentation. VMSS also allows to use automatic OS image upgrade to install Windows patches on custom images.
+3. Move workloads to new node type.
+
+> :exclamation:
+> MCR is currently supported on Service Fabric 9.0 CU2 or later with Windows Server 2022.  
+
+### Scenario 2/Option 1: Full rebuild of Azure Service Fabric cluster (9.0 CU2 or later) on a supported Windows 2022 OS SKU 
+
+This scenario fits where availability loss is acceptable, and effort is less through automation.
+
+Steps
+1. Full rebuild of Service Fabric cluster on a supported OS SKU without container support.
+   Example for the OS SKU name: Windows Server 2022 Datacenter
+2. Re-deploy applications
+
+:exclamation: Please consider the option to recreate the cluster by only removing the Azure Virtual Machine Scale Sets (VMSS) and the Azure Service Fabric cluster resource. Creating just these two instances works well when you don’t automate the whole deployment.
+
+Documentation:
+- [Quickstart: Create a Service Fabric cluster using ARM template](https://docs.microsoft.com/en-us/azure/service-fabric/quickstart-cluster-template)
+- [How To: Rebuild Azure Service Fabric cluster (minimal version)](#)
```

Comment:
invalid link here

Reply:
The link will be added as soon as we have the page.